<launch>
   <node name="Stage" pkg="stage_ros" type="stageros" args="$(find project1)/worlds/mundo.world"/>

   <node name="moveRobot" pkg="project1" type="moveRobot" output="screen">
      <param name="id_robot" value="1" />
      <param name="algor" value="1" />
      <param name="crit_dist" value="0.8" />
      <param name="d_obj" value="0.2" />
      <param name="v_max_des" value="0.8" />
      <param name="v_max_rot" value="0.8" />
      <param name="k_rot_min" value="0.05" />
      <param name="k_rot_max" value="0.2" />
      <param name="ori_error" value="0.3" />
      <param name="t_avoid_obs" value="1.0" />
      <param name="w_1" value="1.0" />
      <param name="w_2" value="2.5" />
      <param name="t_wait" value="100" />
   </node>

   <node name="talkerGoals" pkg="project1" type="talkerGoals" output="screen">
      <param name="x_goal" value="7" />
      <param name="y_goal" value="8" />
      <param name="t_goal" value="2" />
   </node>
</launch>
